front/src/applications/operationalStudies/helpers/TrainOpProjectionLazyLoader.ts

```diff
@@ -37,7 +37,8 @@ export default class TrainOpProjectionLazyLoader extends TrainProjectionLazyLoad
   }
 
   async processBatch(batch: TimetableItemId[]) {
-    const { infraId, path, electricalProfileSetId } = this.options;
+    const { infraId, pathfinding, electricalProfileSetId } = this.options;
+    const path = pathfinding.path;
```

Comment:
I think this can be simplified like that (same if other file)
```suggestion
    const { infraId, pathfinding: { path }, electricalProfileSetId } = this.options;
```